mlrun/common/schemas/model_monitoring/constants.py

```diff
@@ -119,7 +144,8 @@ class FileTargetKind:
     ENDPOINTS = "endpoints"
     EVENTS = "events"
     STREAM = "stream"
-    PARQUET = "parquet"
+    STREAM_PARQUET = "stream_parquet"
```

Comment:
@assaf758 , @theSaarco... not sure if this change will make a  backward compatibility issues, what do you think?